### DISCUSSION TITLE
Should model card be included in gguf files?

### DISCUSSION THREAD

**Discussion by mofosyne** [ggml-org/llama.cpp#8602] [Ideas]
Models in hugging face tends to include a model card.

It's usually a detailed text file with some images. Was wondering if we should include it in the KV store and if so then how? And how do we deal with the stuff around it... like images and the yaml headers?

Some model cards also have image banners and icons. Maybe we want to allow model authors to attach a small image icon so it's easier to find their model if we have a model library?

**Comment by ggerganov** [comment]
Probably a KV pair for model card URL would be sufficient and general enough. It can point to HF model cards for example

**Comment by mofosyne** [comment]
Makes sense, it would keep with the existing pattern of `general.repo_url` for the git repo and `general.url` for stuff like whitepaper/homepage

So perhaps we can go with `general.model_card_url` or direct people to use `general.url` (which would likely just be some documentation update)

For now, just going to be observing how people use `general.url` in the field
